assets/js/blocks/product-categories/hierarchy.js

```diff
@@ -27,8 +23,8 @@ export function buildTermsTree( list = [] ) {
 	const tree = fillWithChildren( termsByParent[ '0' ] || [] );
 	delete termsByParent[ '0' ];
 
-	// anything left in termsByParent has no visible parent
-	forEach( termsByParent, ( terms ) => {
+	// anything left in termsByParent has no visible parent. Native JS instead of lodash forEach.
```

Comment:
I think we can skip this comment: `Native JS instead of lodash forEach.`, in the future I think it might be more confusing than anything else, what do you think?